manage.py

```diff
@@ -407,5 +408,13 @@ def load_efile_sheets():
         df.to_json(path_or_buf="data/" + table + ".json", orient='values')
 
 
+@manager.command
+def slack_message(message):
+    """ Sends a message to the bots channel. you can add this command to ping you when a task is done, etc.
+    run ./manage.py slack_message 'The message you want to post'
+    """
+    post_to_slack(message, '#bots')
```

Comment:
Oh awesome, this is a great idea!  Also, should we set the default channel as an environment variable instead?

Reply:
We opted not to after discussing this more - if we find that this'd be useful we can always do it in the future!